src/ploneintranet/userprofile/browser/configure.zcml

```diff
@@ -0,0 +1,34 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:five="http://namespaces.zope.org/five"
+    xmlns:browser="http://namespaces.zope.org/browser"
+    xmlns:i18n="http://namespaces.zope.org/i18n"
+    xmlns:genericsetup="http://namespaces.zope.org/genericsetup"
+    xmlns:plone="http://namespaces.plone.org/plone"
+    i18n_domain="ploneintranet">
+
+  <browser:page
+      name="userprofile-view"
+      for="ploneintranet.userprofile.content.userprofile.IUserProfile"
+      class=".userprofile.UserProfileView"
+      permission="zope2.View"
+      template="templates/userprofile.pt"
+      />
+
+  <browser:page
+      for="Products.CMFPlone.interfaces.IPloneSiteRoot"
+      name="author"
+      class=".userprofile.AuthorView"
+      permission="zope2.View"
+      layer="ploneintranet.theme.interfaces.IThemeSpecific"
+     />
```

Comment:
That is incorrect. You're not allowed to bind to the theme layer here. You should bind to a userprofile layer.
We want to be able to re-use this package with another theme.